pkg/tsdb/mssql/mssql.go

```diff
@@ -130,6 +131,13 @@ func (t *mssqlQueryResultTransformer) TransformQueryResult(columnTypes []*sql.Co
 				} else {
 					t.log.Debug("Rows", "Error converting numeric to float", value)
 				}
+			case "UNIQUEIDENTIFIER":
+				v, err := uuid.FromBytes(value)
+				if err == nil {
+					values[i] = v.String()
+				} else {
+					t.log.Debug("Rows", "Error converting uniqueidentifier to string", value)
+				}
```

Comment:
After @kylebrandt comment I think we actually can use the uniqueidentifier type in mssql lib to parse it instead of uuid lib. Something along these lines:
```go
uuid := &mssql.UniqueIdentifier{}
if err := uuid.Scan(value); err == nil {
  values[i] = uuid.String()
} else {
  t.log.Debug("Rows", "Error converting uniqueidentifier to string", value)
}
```

Reply:
@marefr I have edited (and tested with my own data) as requested. Thanks for sharing your knowledge.